scripts/asm-installer/tests/setup_longterm_cluster

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+set -CeEu
+set -o pipefail
+
+CLUSTER_NAME="long-term-test-cluster"
```

Comment:
should we create long term cluster for each branch? Istio revision may share some base resources within the same clusters, and a new minor version istiod may stop working on old base resources.

Reply:
That's a good catch. Although now I wonder if we should just keep one cluster in order to catch the case where that happens, since I bet we don't have a great idea when it does.